fluentd/Gemfile

```diff
@@ -1,14 +1,14 @@
 gem 'fluentd', ENV['FLUENTD_VERSION']
```

Comment:
please remove the env var here and from the Dockerfiles and we'll peg the version in the Gemfile too

Reply:
AFAICS, `ENV['FLUENTD_VERSION']` is used heavily in the source code. I hesitate to delete this. We should keep the version pin in the dockerfile. WDYT?